edgelet/iotedge/src/config/apply.rs

```diff
@@ -14,7 +14,7 @@ const AZIOT_EDGED_HOMEDIR_PATH: &str = "/var/lib/aziot/edged";
 const TRUST_BUNDLE_USER_ALIAS: &str = "trust-bundle-user";
 
 // TODO: Dedupe this with edgelet-http-workload
```

Comment:
Is this TODO still relevant?

Reply:
https://github.com/Azure/iotedge/pull/5998/files#diff-c82483c8507fde57eac9b98efc5b81c24d7f71414667f0d10439f28c421e9e30R250